src/domain/wallets/tx-history.ts

```diff
@@ -317,7 +337,9 @@ const displayCurrencyPerBaseUnitFromAmounts = ({
 }: {
   displayAmountAsNumber: number
   settlementAmountInBaseAsNumber: number
-}): number =>
-  settlementAmountInBaseAsNumber === 0
+}): number => {
+  const usdDisplayAmountAsNumber = Number((displayAmountAsNumber / 100).toFixed(2))
```

Comment:
why to rename it to usd?

Reply:
Fixed here, I was trying to capture that it wasn't "cents" unit https://github.com/GaloyMoney/galoy/pull/2191/commits/44b0e9871afdd7cdabf79be770117e1af72d6118